website/pages/docs/k8s/helm.mdx

```diff
@@ -86,7 +86,7 @@ and consider if they're appropriate for your deployment.
   - `acls` ((#v-global-acls)) - Configure ACLs.
 
     - `manageSystemACLs` ((#v-global-acls-managesystemacls)) (`boolean: false`) - If true, the Helm chart will automatically manage ACL tokens and policies for all Consul and consul-k8s components.
-      This requires servers to be running inside Kubernetes. Additionally requires Consul >= 1.4 and consul-k8s >= 0.14.0.
+      This requires Consul >= 1.4 and consul-k8s >= 0.14.0.
```

Comment:
This is more for my own clarification, but how do we treat capitalization? `Consul` vs `consul`? `Consul-K8s` vs `consul-k8s`?

Reply:
I think I typically use Consul and consul-k8s. The reason is Consul refers to a product, while consul-k8s refers to a repo rather than a product.